public/app/plugins/panel/nodeGraph/panelcfg.cue

```diff
@@ -43,12 +43,12 @@ composableKinds: PanelCfg: {
 					// Unit for the secondary stat to override what ever is set in the data frame.
 					secondaryStatUnit?: string
 				}
-        ZoomMode: "cooperative" | "greedy" @cuetsy(kind="enum")
```

Comment:
these changes were applied after merging from main, probably a change in the format?